frontend/require/NavView.js

```diff
@@ -3,7 +3,7 @@ define(['underscore', 'backbone', 'mustache', 'text!NavView.html'], function(_,
 
     var NavView = Backbone.View.extend({
 
-        tagName: 'nav',
+        tagName: 'div',
```

Comment:
It probably doesn't matter... it just seemed weird to have a `<nav>` nested in a `<nav>`